### PR TITLE
Allow loopback addresses for redirect uris

### PR DESCRIPTION
**Pull request by MKodde**
This validation was added to the parsley validator. This should be sufficient validation. As when Javascript is disabled you wouldnt be able to submit the form with redirect uris.

See https://www.pivotaltracker.com/story/show/171620767

### CODE REVIEW

File: src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Form/Entity/OidcngEntityType.php [hunk 1]
```diff
@@ -111,7 +111,6 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'entry_options' => [
                         'attr' => [
                             'data-parsley-redirecturis' => null,
-                            'data-parsley-uri' => null,
```
Comment (pablothedude) [inline src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Form/Entity/OidcngEntityType.php:114]: Should this also be removed in the OidcEntity ?

File: app/js/components/validator_helper.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+export class ValidatorHelper{
```
Comment (pablothedude) [inline app/js/components/validator_helper.ts:1]: I'm missing type hinting for return types on all methods

File: src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Form/Entity/OidcngEntityType.php (hunk 1 above)
Comment (MKodde) [reply src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Form/Entity/OidcngEntityType.php:114]: Yes, good point!

File: app/js/components/validator_helper.test.ts [hunk 3]
```diff
@@ -0,0 +1,91 @@
+import { ValidatorHelper } from './validator_helper';
```
Comment (pablothedude) [inline app/js/components/validator_helper.test.ts:1]: Thanks for covering the tests!

**Pull request merged**